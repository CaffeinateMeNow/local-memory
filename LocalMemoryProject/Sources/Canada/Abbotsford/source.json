{"state-name": "", "newspaper": [{"facebook": "http://www.facebook.com/myabbynews?ref=ts&sk=app_7146470109#!/myabbynews", "city-county-lat": 49.0504, "media-subclass": "city-county", "website": "http://www.abbynews.com/", "city-county-name": "Abbotsford", "name": "Abbotsford News", "rss": ["http://www.abbynews.com/opinion/index.rss", "http://www.abbynews.com/opinion/letters/index.rss", "http://www.abbynews.com/community/index.rss", "http://www.abbynews.com/news/index.rss", "http://www.abbynews.com/sports/index.rss", "http://www.abbynews.com/entertainment/index.rss", "http://www.abbynews.com/lifestyles/index.rss", "http://www.abbynews.com/business/index.rss"], "city-county-long": -122.3045, "video": "", "media-class": "newspaper", "twitter": "http://twitter.com/#!/abbynews", "open-search": []}, {"facebook": "", "city-county-lat": 49.0504, "media-subclass": "city-county", "website": "http://punjabipatrika.ca/", "city-county-name": "Abbotsford", "name": "Punjabi Patrika", "rss": ["http://www.patrika.ca/comments/feed/", "http://www.patrika.ca/feed/"], "city-county-long": -122.3045, "video": "", "media-class": "newspaper", "twitter": "", "open-search": []}], "radio": [], "tv": []}