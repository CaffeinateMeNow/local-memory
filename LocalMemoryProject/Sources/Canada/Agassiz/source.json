{"state-name": "", "newspaper": [{"facebook": "http://www.facebook.com/pages/BCLocalNewscom/212373115244#!/AgassizHarrisonObserver", "city-county-lat": 49.2389, "media-subclass": "city-county", "website": "http://www.agassizharrisonobserver.com/", "city-county-name": "Agassiz", "name": "Agassiz Harrison Observer", "rss": ["http://www.agassizharrisonobserver.com/opinion/index.rss", "http://www.agassizharrisonobserver.com/opinion/letters/index.rss", "http://www.agassizharrisonobserver.com/community/index.rss", "http://www.agassizharrisonobserver.com/news/index.rss", "http://www.agassizharrisonobserver.com/sports/index.rss", "http://www.agassizharrisonobserver.com/entertainment/index.rss", "http://www.agassizharrisonobserver.com/lifestyles/index.rss", "http://www.agassizharrisonobserver.com/business/index.rss"], "city-county-long": -121.7659, "video": "", "media-class": "newspaper", "twitter": "http://twitter.com/#!/agassizobserver", "open-search": []}], "radio": [], "tv": []}