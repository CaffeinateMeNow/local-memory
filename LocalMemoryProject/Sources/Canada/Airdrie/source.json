{"state-name": "", "newspaper": [{"facebook": "https://www.facebook.com/airdriecityviewnews", "city-county-lat": 51.2917, "media-subclass": "city-county", "website": "http://www.airdriecityview.com/", "city-county-name": "Airdrie", "name": "Airdrie City View", "rss": [], "city-county-long": -114.0144, "video": "", "media-class": "newspaper", "twitter": "http://twitter.com/RV_Publishing", "open-search": []}, {"facebook": "https://www.facebook.com/pages/Airdrie-Echo/140376026026189", "city-county-lat": 51.2917, "media-subclass": "city-county", "website": "http://www.airdrieecho.com", "city-county-name": "Airdrie", "name": "Airdrie Echo", "rss": [], "city-county-long": -114.0144, "video": "", "media-class": "newspaper", "twitter": "http://www.twitter.com/Airdrie_Echo", "open-search": []}, {"facebook": "https://www.facebook.com/heresthescoop?ref=hl", "city-county-lat": 51.2917, "media-subclass": "city-county", "website": "http://www.heresthescoop.com/", "city-county-name": "Airdrie", "name": "Here's The Scoop", "rss": ["http://www.heresthescoop.com/?format=feed&type=atom", "http://www.heresthescoop.com/?format=feed&type=rss"], "city-county-long": -114.0144, "video": "", "media-class": "newspaper", "twitter": "https://twitter.com/search?q=airdriescoop&src=savs", "open-search": ["http://www.heresthescoop.com/component/search/?Itemid=148&format=opensearch"]}], "radio": [], "tv": []}