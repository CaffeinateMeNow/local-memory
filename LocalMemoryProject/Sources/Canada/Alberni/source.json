{"state-name": "", "newspaper": [{"facebook": "", "city-county-lat": 49.2339, "media-subclass": "city-county", "website": "http://www2.canada.com/albernivalleytimes/", "city-county-name": "Alberni", "name": "Alberni Valley Times", "rss": [], "city-county-long": -124.8055, "video": "", "media-class": "newspaper", "twitter": "", "open-search": []}], "radio": [], "tv": []}