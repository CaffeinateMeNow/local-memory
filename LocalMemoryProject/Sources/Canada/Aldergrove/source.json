{"state-name": "", "newspaper": [{"facebook": "http://www.facebook.com/pages/Aldergrove-Star/186177368124190", "city-county-lat": 49.0581, "media-subclass": "city-county", "website": "http://www.aldergrovestar.com/", "city-county-name": "Aldergrove", "name": "Aldergrove Star", "rss": ["http://www.aldergrovestar.com/opinion/index.rss", "http://www.aldergrovestar.com/opinion/letters/index.rss", "http://www.aldergrovestar.com/community/index.rss", "http://www.aldergrovestar.com/news/index.rss", "http://www.aldergrovestar.com/sports/index.rss", "http://www.aldergrovestar.com/entertainment/index.rss", "http://www.aldergrovestar.com/lifestyles/index.rss", "http://www.aldergrovestar.com/business/index.rss"], "city-county-long": -122.4707, "video": "", "media-class": "newspaper", "twitter": "http://twitter.com/#!/aldergrovestar", "open-search": []}], "radio": [], "tv": []}