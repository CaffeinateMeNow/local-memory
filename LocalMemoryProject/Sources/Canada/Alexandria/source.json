{"state-name": "", "newspaper": [{"facebook": "", "city-county-lat": 45.3112, "media-subclass": "city-county", "website": "http://www.glengarrynews.ca/", "city-county-name": "Alexandria", "name": "Glengarry News", "rss": ["http://www.glengarrynews.ca/rss.xml"], "city-county-long": -74.6367, "video": "", "media-class": "newspaper", "twitter": "", "open-search": []}], "radio": [], "tv": []}