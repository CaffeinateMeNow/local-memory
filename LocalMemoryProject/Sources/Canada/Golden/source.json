{"state-name": "", "newspaper": [{"facebook": "http://www.facebook.com/pages/The-Golden-Star/146914265388935", "city-county-lat": 51.2961, "media-subclass": "city-county", "website": "http://www.thegoldenstar.net/", "city-county-name": "Golden", "name": "Golden Star", "rss": ["http://www.thegoldenstar.net/opinion/index.rss", "http://www.thegoldenstar.net/opinion/letters/index.rss", "http://www.thegoldenstar.net/community/index.rss", "http://www.thegoldenstar.net/news/index.rss", "http://www.thegoldenstar.net/sports/index.rss", "http://www.thegoldenstar.net/entertainment/index.rss", "http://www.thegoldenstar.net/lifestyles/index.rss", "http://www.thegoldenstar.net/business/index.rss"], "city-county-long": -116.9631, "video": "", "media-class": "newspaper", "twitter": "http://twitter.com/#!/goldenstarnews", "open-search": []}], "radio": [], "tv": []}